src/routes.tsx

```diff
@@ -36,269 +38,331 @@ enum EmbarkStory {
+                        : EmbarkStory.NorwayComboEnglish,
+                  }
+              }
+              break
+            case 'se':
+            case 'se-en':
+              switch (name) {
+                case 'new':
+                  return {
+                    baseUrl: `/${locale}/new-member/new`,
+                    name: EmbarkStory.SwedenNeeder,
+                  }
+                case 'switch':
+                  return {
+                    baseUrl: `/${locale}/new-member/switch`,
+                    name: EmbarkStory.SwedenSwitcherWithoutAccident,
+                  }
+                case 'home-accident-switcher':
+                  if (
+                    window.hedvigClientConfig.appEnvironment === 'production'
+                  ) {
+                    return {
+                      redirect: `/${locale}/new-member`,
+                    }
+                  }
+                  return {
+                    baseUrl: `/${locale}/new-member/home-accident-switcher`,
+                    name: EmbarkStory.SwedenSwitcher,
+                  }
+                case 'home-accident-needer':
+                  if (
+                    window.hedvigClientConfig.appEnvironment === 'production'
+                  ) {
+                    return {
+                      redirect: `/${locale}/new-member`,
+                    }
+                  }
+                  return {
+                    baseUrl: `/${locale}/new-member/home-accident-needer`,
+                    name: EmbarkStory.SwedenQuoteCartNeeder,
+                  }
+              }
+              break
+          }
 
-        return {
-          redirect: `/${locale}/new-member`,
+          return {
+            redirect: `/${locale}/new-member`,
+          }
         }
-      }
 
-      const props = getProps()
+        const props = getProps()
 
-      if (props.redirect) {
-        return <Redirect to={props.redirect} />
-      }
+        if (props.redirect) {
+          return <Redirect to={props.redirect} />
+        }
 
-      return (
-        <EmbarkRoot
-          language={match.params.locale}
-          name={props.name}
-          baseUrl={props.baseUrl}
-        />
-      )
-    },
-    exact: false,
+        return (
+          <EmbarkRoot
+            language={match.params.locale}
+            name={props.name}
+            baseUrl={props.baseUrl}
+          />
+        )
+      },
+      exact: false,
+    },
+  },
+  {
+    /* 
+    This route is only used as a web view in the apps
+    for markets using Simple Sign, since we're missing some certificates
+    */
+    path: localePathPattern + '/login',
+    serverRouteData: {
+      titleTextKey: 'Text',
+    },
+    clientRouteData: {
+      Component: LoginApp,
+      exact: true,
+    },
+  },
+  {
+    path: /^(?!\/?new-member-assets).*$/,
+    serverRouteData: {
+      titleTextKey: 'FOUR_OH_FOUR_PAGE_TITLE',
+      status: 404,
+    },
   },
   {
     path: '/*',
```

Comment:
Shouldn't we use this `serverRouteData` here?

```
serverRouteData: {
       titleTextKey: 'FOUR_OH_FOUR_PAGE_TITLE',
       status: 404,
     },
```

Reply:
For the `/*` paths? The way it is currently is that this one is only part of the `reactPageRoutes` array. And then the server route instead uses the regex `/^(?!\/?new-member-assets).*$/` to set the status to `404` for unknown paths. So I don't think we need to add it again. But maybe I'm misunderstanding. 

Also I'm not entirely sure why this is even done this way in the first place haha 🙈 Apart from the fact that we don't want a `404` status when requesting images from **https://www.hedvig.com/new-member-assets**, like [https://www.hedvig.com/new-member-assets/landing/hedvig_table_landscape_medium.jpg](https://www.hedvig.com/new-member-assets/landing/hedvig_table_landscape_medium.jpg)

Reply:
Hmm okey, yeah I guess it's fine then :)